frontend/src/components/FloatingElement/FloatingElement.tsx

```diff
@@ -7,23 +7,26 @@ import useMediaQuery from '@mui/material/useMediaQuery'
 interface FloatingElementProps {
   children: React.ReactNode
   anchorEl: HTMLButtonElement | null
+  subscribedFromEmail: boolean
```

Comment:
Change naming. If the value of a variable is boolean, the naming should start with a verb, i.e. isSubscribedFromEmail, hasSubscription, isClickedFromEmail. 

Also, hindi pa ito subscribed tama ba? subscribing palang? baka pwedeng iconsider ang `isOpenedFromEmail` or something similar.